packages/cli/generators/relation/has-many-relation.generator.js

```diff
@@ -53,6 +53,35 @@ module.exports = class HasManyRelationGenerator extends BaseRelationGenerator {
       options.destinationModelPrimaryKey;
     this.artifactInfo.foreignKeyName = options.foreignKeyName;
 
+    const project = new relationUtils.AstLoopBackProject();
+    const sourceFile = relationUtils.addFileToProject(
+      project,
+      this.artifactInfo.modelDir,
+      options.sourceModel,
+    );
+    this.artifactInfo.exclude = [this.artifactInfo.targetModelPrimaryKey];
+    const sourceClass = relationUtils.getClassObj(
+      sourceFile,
+      options.sourceModel,
+    );
+    for (const classProperty of sourceClass.getInstanceProperties()) {
```

Comment:
This code seems to be repetitive. Can we move it to some common place and use it at multiple places ?

Reply:
Moved the code to relation utils.